### PR TITLE
Plot loading animations and removing EigenValue plot legend

### PR DESCRIPTION
**Pull request by sandertyu**
This pull request includes loading animations for the plots, they now have a fixed height so the screen stays in place while loading, and I added a keyword argument to the plot_eigenvalues_vs_speed() function in main.py allowing us to remove the legend from displaying in the app.

### CODE REVIEW

File: bicycleparameters/main.py [hunk 1]
```diff
@@ -896,7 +896,7 @@ def eig(self, speeds):
 
     def plot_eigenvalues_vs_speed(self, speeds, fig=None, generic=False,
                                   color='black', show=False, largest=False,
-                                  linestyle='-', grid=False):
+                                  linestyle='-', grid=False, showLegend=True):
```
Comment (moorepants) [inline bicycleparameters/main.py:899]: Let's make this `show_legend` to follow PEP8. I wrote much of this code before I knew about PEP8 and new things should adopted PEP8. I'll eventually update all the code to follow.

Comment (moorepants) [reply bicycleparameters/main.py:899]: Be sure to add this in the docstring below too.

Comment (sandertyu) [reply bicycleparameters/main.py]: Okay, by docstring did you mean the if statement? I think I have everything corrected now.

Comment (sandertyu) [reply bicycleparameters/main.py]: Yeah that's what it is, I went ahead and added my own description there, go ahead and change it if you would like.

**Comment by sandertyu** [comment]
I have pushed the docstring update to [my branch on origin](https://github.com/sandertyu/BicycleParameters/commits/Loading-and-Legend) and the commit shows up there, but it is not appearing here in this pull request. Not really sure why that is.

File: bicycleparameters/main.py (hunk 1 above)
Comment (moorepants) [reply bicycleparameters/main.py]: > The comment area below the function explaining the parameters?

Yes.

**Comment by moorepants** [comment]
Not sure either, seems like a github issue. You can commit and empty commit and push to see if it triggers things.

https://coderwall.com/p/vkdekq/git-commit-allow-empty

**Comment by sandertyu** [comment]
Okay, it's still not showing up here in the pull request but under commits there is one with the message "++show_legend to docstring", and it is in the commits for this pull request now. I believe everything is ready to merge.

**Pull request merged**